server/pageRenderer.js

```diff
@@ -112,8 +112,7 @@ export default function pageRenderer({
 
         <link href="https://cdn.jsdelivr.net/npm/font-awesome@4.7.0/css/font-awesome.min.css" rel="stylesheet">
         <link href="https://unpkg.com/ionicons@3.0.0/dist/css/ionicons.min.css" rel="stylesheet">
-        <link href="https://fonts.googleapis.com/css?family=Open+Sans:700|Raleway|Roboto" rel="stylesheet">
-        ${links}
```

Comment:
Why remove this?

Reply:
Nothing has been removed, just forgotten to push 🙃 Open Sans behaves a bit weird, as it requires us to specify the weight of the font to be imported (as oppose to Roboto and Raleway)